src/client/components/pages/search.js

```diff
@@ -43,18 +39,15 @@ class SearchPage extends React.Component {
 
 		this.state = {
 			from: 0,
-			nextEnabled: this.props.initialResults.length === this.props.resultsPerPage,
 			query: this.props.query,
 			results: this.props.initialResults,
 			size: this.props.resultsPerPage
```

Comment:
I think for any page that uses the PagerElements, you don't need to store the `size` in the state. You can pass it directly to PagerElement as `this.props.size`.

Reply:
And I think the same is true for the `from` property. It should only change when `triggerSearch` is called and the query has changed.

Reply:
I'm on my phone right now. I'll have a look at this in the evening or maybe tomorrow. I've got classes right now :P